src/_playground/App.scss

```diff
@@ -3,6 +3,7 @@ $fd-fonts-path: '../../node_modules/fiori-fundamentals/dist/fonts/';
 @import '../../node_modules/fiori-fundamentals/scss/all.scss';
 
 // Fundamental React Docs Styles
+
```

Comment:
@leannerivera - first off congrats on your first PR!

There are some unrelated changes here that should be removed from this PR so that it remains focused.

In App.scss there are some white space diffs that should be removed.